core/arch/arm/include/mm/tee_pager.h

```diff
@@ -110,7 +110,16 @@ void tee_pager_add_core_area(vaddr_t base, size_t size, uint32_t flags,
  *
  * Return true on success of false if the area can't be added
  */
+#ifdef CFG_PAGED_USER_TA
 bool tee_pager_add_uta_area(struct user_ta_ctx *utc, vaddr_t base, size_t size);
+#else
+static inline bool tee_pager_add_uta_area(struct user_ta_ctx *utc __unused,
+					  vaddr_t base __unused,
+					  size_t size __unused)
+{
+	return false;
```

Comment:
Calling this function when `CFG_PAGED_USER_TA` is disabled should never happen and would denote a memory corruption case. `panic()` ?

Reply:
Something is wrong if this function is called, but adding a `panic()` seems a bit excessive.